app/src/components/fields/StartEndDateFields.tsx

```diff
@@ -25,6 +25,7 @@ const StartEndDateFields: React.FC<IStartEndDateFieldsProps> = (props) => {
         <TextField
           id="start_date"
           name="start_date"
+          data-testid="start-date"
```

Comment:
question:
why did this get added? I dont see it being used in any of the tests

Reply:
see my previous comment